### ISSUE TITLE
Alert and AlertGroup component test page

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#1253]
To test that alerts work as intended, we need a test page that does the following:

1. automatically insert Alert 1 ~5 seconds after page load
2. automatically insert Alert 2 ~3 seconds after Alert 1
3. automatically dismiss each alert after ~8 seconds
4. include a form, where on submission, an alert displays
    - After the form alert displays, repeat steps 1-3, except that only alerts 1 and 2 are automatically dismissed. 

Proposed structure would be:
- have an ever-present `<div>` with `role="status"` and `aria-atomic` where alerts 1 and 2 would be inserted
- have an ever-present `<div>` with `role="alert"`(?) and `aria-atomic`(?) where alert 3 would be inserted
- Alert body would roughly be like this:
   ```
    <h4 class="pf-c-alert__title">
      <span class="pf-screen-reader">Success alert:</span>
      Success notification title that is really long whoa
    </h4>
    <p class="pf-c-alert__content">Success notification description that is really long y. <a href="#">This is a link.</a></p>
    <div class="pf-c-alert__action">
      <button class="pf-c-button pf-m-plain pf-m-close" aria-label="Close Success Alert: Success notification title">...</button>
   ```

Things we are testing for:
- Will alert 1 be fully announced before alert 2 is fully announced?
- How are alerts announced when the form is submitted?

As part of this issue, we're hoping to finalize whether we should use `role="status"` vs `role="alert"` for each context. And also if `aria-atomic` affects the experience, and if so, what value should be set.

**Comment by jgiardino** [comment]
@seanforyou23 You seemed to have a good understanding of what the `aria-atomic` value should be, so I left that for you to figure out and provide a recommendation on. I'm not really sure if that's needed on the form alert or now.

**Comment by jgiardino** [comment]
Another attribute worth testing for this is `aria-relevant="additions"`.

An example of this is included here: https://inclusive-components.design/notifications/#conversations

**Comment by jgiardino** [comment]
The findings from the POC created for this issue are:

- These attributes work best for the live region for the type of dynamic notifications we are displaying:
  ```
  <div class="live-region-container"
    aria-atomic="false"
    aria-live="polite"
    aria-relevant="additions text">
  </div>
  ```
- This live region must exist in the DOM 200ms before the message is inserted. This is required for JAWS and NVDA. VO however will announce the contents of the live region when inserted at the same time.
